README.md

```diff
@@ -55,6 +55,24 @@ wasmer run examples/sqlite.wasm
 wasmer run examples/nginx/nginx.wasm -- -p examples/nginx -c nginx.conf
 ```
 
+#### With WAPM
+
+Installing Wasmer through `get.wasmer.io` includes
+[wapm](https://github.com/wasmerio/wapm-cli), the WebAssembly package manager.
+
+Wapm allows you to easily download, run, and distribute WebAssembly binaries.
+
+```sh
+# Install cowsay globally
+wapm install cowsay
```

Comment:
Is this command for installing it locally, instead of globally?

Reply:
Good catch!